Docs/developer_guide/build_instructions/macos.md

```diff
@@ -5,74 +5,108 @@
 The prerequisites listed below are required to be able to configure/build/package/test Slicer.
 
 - XCode command line tools must be installed:
-```
+
+```bash
 xcode-select --install
 ```
+
 - A CMake version that meets at least the minimum required CMake version [here](https://github.com/Slicer/Slicer/blob/master/CMakeLists.txt#L1)
-- Large File Storage for git is required. (`brew install git-lvs`)
+- Large File Storage for git is required. (`brew install git-lfs`)
```

Comment:
We should double check but I think this is not required anymore.

Reply:
I agree, I'm pretty sure I didn't install it in my latest mac and it doesn't show up in `brew list`